mozilla/persona
# Prevent the "unknown path" console message from being displayed on the "/tos" and "/privacy" pages.

To test:
1) Open your dev console
2) go to the main site's /tos and /privacy pages.
3) Check for "unknown path" in the console.

issue #1518
